### PR TITLE
chore: bump cli versions

### PR DESCRIPTION
**Pull request by sobolk**
#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] Any CDK or CloudFormation parameter changes are called out explicitly

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
     "update-versions": "lerna version --yes --no-commit-hooks --no-push --exact --conventional-commits --no-git-tag-version",
     "commit": "git-cz",
     "coverage": "codecov || exit 0",
-    "add-cli-no-save": "yarn add @aws-amplify/cli-internal@^11.0.0-zzz-beta.0 -W  && git checkout -- package.json",
+    "add-cli-no-save": "yarn add @aws-amplify/cli-internal@^11.0.0 -W  && git checkout -- package.json",
```
Comment (alharris-at) [inline package.json:42]: Probably don't need the qualifier here, but I'm fine leaving it to make it explicit.

**Review by alharris-at — APPROVED** [review]

**Review by evcodes — DISMISSED** [review]

File: .circleci/config.base.yml [hunk 2]
```diff
@@ -51,7 +51,7 @@ install_cli_from_local_registry: &install_cli_from_local_registry
     startLocalRegistry "$(pwd)/.circleci/verdaccio.yaml"
     setNpmRegistryUrlToLocal
     changeNpmGlobalPath
-    npm install -g @aws-amplify/cli-internal@^11.0.0-zzz-beta.0
+    npm install -g @aws-amplify/cli-internal@^11.0.0
```
Comment (phani-srikar) [inline .circleci/config.base.yml:54]: can we just use whatever is the latest available and remove `@^11.0.0`? Saves us from keep updating this file.

File: .circleci/config.yml [hunk 3]
```diff
@@ -35,7 +35,7 @@ install_cli_from_local_registry: &install_cli_from_local_registry
     startLocalRegistry "$(pwd)/.circleci/verdaccio.yaml"
     setNpmRegistryUrlToLocal
     changeNpmGlobalPath
-    npm install -g @aws-amplify/cli-internal@^11.0.0-zzz-beta.0
+    npm install -g @aws-amplify/cli-internal@^11.0.0
```
Comment (phani-srikar) [inline .circleci/config.yml:38]: same here

**Review by phani-srikar — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (sobolk) [reply package.json:42]: Unless we rollback :D
Making it explicit at least revels this place if somebody is full text searching for versions.

File: .circleci/config.base.yml (hunk 2 above)
Comment (sobolk) [reply .circleci/config.base.yml:54]: That matches all `11.xx.yy` versions, so you'll get latest until we ship major version. See also comments above https://github.com/aws-amplify/amplify-category-api/pull/1339#discussion_r1134480879 .

I'm fine either way you and @alharris-at decide.

Comment (sobolk) [reply .circleci/config.base.yml]: removed.

**Review by evcodes — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (sobolk) [reply package.json]: I removed it per @phani-srikar request.

**Review by phani-srikar — APPROVED** [review]

**Pull request merged**